CHANGES.txt

```diff
@@ -13,6 +13,11 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Fix reproducible builds. Restore logic respecting SOURCE_DATE_EPOCH when set.
     - Fix version tests to work with updated scons --version output. (Date format changed)
 
+  From Daniel Moody:
+    - Fix command line escaping for ninja dollar sign escape. Without escaping ninja properly,
+      the ninja file scons regenerate and callback invocations will lose the $ characters used in
+      the scons command line which ninja uses itself for escaping.
+
```

Comment:
Can you be more specific about what's fixed here? I had to carefully examine the new test to figure it out.

Reply:
@bdbaddog updated with specific example: 
9ac2aa4